webapp/public/run.html

```diff
@@ -166,6 +167,10 @@
                     // propagate message packets to parent frames
                     window.parent.postMessage(msg.data, "*");
                 }
+                else if (mpRole && data.type === "multiplayer") {
+                    // propagate message packets to parent frames
+                    window.parent.postMessage(msg.data, "*");
```

Comment:
@jwunderl I'm seeing incorrect behavior here in production (beta). But I think it makes sense. This issue is that the client isn't rendering the screen. What I believe is happening: on the client, this method receives a "broadcast-screen" message, passed to this frame from the parent. And what we do here it pass it right back to the parent! I can confirm it is getting received by the client parent. I wasn't seeing this behavior when hosting locally, for reasons unknown.

My understanding is this method can receive messages from the child iframe or the parent. I think the solution is that we need to know which direction to route the message. Does that sound right to you? Do we do this elsewhere?

Reply:
Made a fix here: https://github.com/microsoft/pxt/pull/8814